i18n/checkLocaleKeys.js

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs'
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+// Convert import.meta.url to __dirname equivalent
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+
+// Define the path to our i18n locales directory
+const localesI18nDirectory = path.join(
+    __dirname,
+    '/locales'
+)
+
+// Read all JSON files from the locales directory
+let localeFiles = fs
+    .readdirSync(localesI18nDirectory)
+    .filter(file => file.endsWith('.json'))
+
+// Ensure en.json is the first file
+const enFile = 'en.json'
+if (!localeFiles.includes(enFile)) {
+    console.error('en.json file not found in the locales directory')
+    process.exit(1)
+}
+localeFiles = localeFiles.filter(file => file !== enFile)
+localeFiles.unshift(enFile)
```

Comment:
instead of unshifting, let's just keep the `enFile` as a separate variable to make it more readable